inst/include/generated/state_types.hpp

```diff
@@ -145,8 +145,7 @@ struct ChildModelState<ChildModel, real_type> {
     const real_type hiv_births,
     const TensorFixedSize<real_type, Sizes<hcAG_coarse<ChildModel>>>& hc_art_init,
     const TensorFixedSize<real_type, Sizes<hc1DS<ChildModel>, hcTT<ChildModel>, hcAG_end<ChildModel>, NS<ChildModel>>>& hc_art_need_init,
-    const real_type ctx_need,
-    const real_type ctx_mean
+    const real_type ctx_need
```

Comment:
Can you add `infection_by_type` here too please? This won't be affecting the R only the Python interface atm

`const TensorFixedSize<real_type, Sizes<hcTT<ChildModel>, hc1AG<ChildModel>, NS<ChildModel>>>& infection_by_type`

And below in the initializer list

Reply:
Mantra has said he will add that in when he integrates the branches.